packages/serverpod/test/util/path_util.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:serverpod/src/util/path_util.dart';
+import 'package:test/test.dart';
+import 'package:path/path.dart' as p;
+
+void main() {
```

Comment:
Chore: Add `_test.dart` suffix to file name.

This ensures that the dart test language server can understand the file correctly.